docs/jade-getting-started.md

```diff
@@ -363,43 +313,89 @@ export AZURE_SYNAPSE_SQLADMINPASSWORD=$(cat /tmp/jade-dev-synapse-admin-password
 export AZURE_SYNAPSE_ENCRIPTIONKEY=$(cat /tmp/jade-dev-synapse-encryption-key.key)
 export AZURE_SYNAPSE_INITIALIZE=false
 
+# RBS
+# Defaults to RBS tools; you can switch to dev by running `./render-configs.sh -r dev`
+export RBS_POOLID=$(cat /tmp/rbs-pool-id.txt)
+export RBS_INSTANCEURL=$(cat /tmp/rbs-instance-url.txt)
+
 # Pact contract test settings
 export PACT_BROKER_USERNAME=$(cat /tmp/pact-ro-username.key)
 export PACT_BROKER_PASSWORD=$(cat /tmp/pact-ro-password.key)
 
-# Azure B2C authentication settings
-export OIDC_ADDCLIENTIDTOSCOPE=true
-export OIDC_AUTHORITYENDPOINT="https://oauth-proxy.dsp-eng-tools.broadinstitute.org/b2c"
-export OIDC_CLIENTID=bbd07d43-01cb-4b69-8fd0-5746d9a5c9fe
-export OIDC_EXTRAAUTHPARAMS="prompt=login"
-export OIDC_PROFILEPARAM=b2c_1a_signup_signin_tdr_dev
```

Comment:
These settings cause our tests to fail. I've moved them to application.properties and overwritten them in our test properties to be empty.

Reply:
I haven't been able to reproduce this error. But, I've still moved them to application properties so that we don't need to manually set them.